Assets/_Laz/Tests/PlanetoidTests.cs

```diff
@@ -58,6 +58,96 @@ public void Test_Planetoid_Reset()
             Assert.AreEqual(mockPlanetoidBehaviour.ResetCount, 1, "Reset should be called once");
         }
 
+        [Test]
```

Comment:
Since you're testing `LazoWrappableManager`, You should create a new file and place the tests there.